lib/bas/bot/write_media_review_in_discord.rb

```diff
@@ -58,17 +54,15 @@ def read
       reader.execute
     end
 
-    # process function to execute the Notion utility to send formated blocks to a page
+    # process function to execute the Discord utility to send image feedback to a thread of Discord channel
```

Comment:
What do you think about changing it to `process function to execute the Discord utility to send image feedback to a thread of a Discord channel.`? :thinking: